qiling/core.py

```diff
@@ -50,6 +50,7 @@ def __init__(
             stdout=0,
             stderr=0,
             output=None,
+            syscall_filter=None,
```

Comment:
since this is during execution, we don't have to add in the init, just self.syscall_filter = None and at line 64 will be good